.github/actions/run-dagger-pipeline/action.yml

```diff
@@ -96,13 +92,6 @@ runs:
       shell: bash
       run: |
         export _EXPERIMENTAL_DAGGER_RUNNER_HOST="unix:///var/run/buildkit/buildkitd.sock"
-        DAGGER_CLI_COMMIT=${{ inputs.dagger_cli_commit }}
-        DAGGER_TMP_BINDIR="/tmp/dagger_${DAGGER_CLI_COMMIT}"
-        export _EXPERIMENTAL_DAGGER_CLI_BIN="$DAGGER_TMP_BINDIR/dagger"
-        if [ ! -f  "$_EXPERIMENTAL_DAGGER_CLI_BIN" ]; then
-          mkdir -p "$DAGGER_TMP_BINDIR"
-          curl "https://dl.dagger.io/dagger/main/${DAGGER_CLI_COMMIT}/dagger_${DAGGER_CLI_COMMIT}_$(uname -s | tr A-Z a-z)_$(uname -m | sed s/x86_64/amd64/).tar.gz" | tar xvz -C "$DAGGER_TMP_BINDIR"
-        fi
```

Comment:
These lines were there to install a custom Dagger CLI. We don't wan't to use a custom Dagger CLI anymore and upgrade to the latest CLI released with 0.6.4